knexfile.js

```diff
@@ -0,0 +1,51 @@
+// Update with your config settings.
+
+module.exports = {
```

Comment:
What is the use case for the staging and production in the knexfile? What is a good example for both?